java/arcs/tools/BUILD

```diff
@@ -50,3 +50,25 @@ java_binary(
     main_class = "arcs.tools.VerifyPolicyKt",
     runtime_deps = [":verify_policy_lib"],
 )
+
+kt_jvm_library(
+    name = "generator",
+    srcs = [
+        "Recipe2Plan.kt",
+    ],
+    deps = [
+        "//java/arcs/core/data",
+        "//java/arcs/core/data/proto:manifest_java_proto",
+        "//java/arcs/core/storage",
+        "//java/arcs/core/storage/api",
+        "//third_party/java/arcs/deps:protobuf_java",
+        "//third_party/kotlin/clikt",
+        "//third_party/kotlin/kotlinpoet",
+    ],
+)
+
+java_binary(
+    name = "recipe2plan_binary",
```

Comment:
Could we adopt some consistent naming for these BUILD rules? 

I'm partial to the naming strategy I used for `verify_policy`, where the binary is given the "correct" name, and the library is given an _lib suffix:

```
kt_jvm_library(
  name = "foo_lib",
  srcs = ["Foo.kt"],
  ...
)

java_binary(
  name = "foo",
  runtimes_deps = [":foo_lib"],
)
```

but honestly anything is fine as long as it is consistent.

Reply:
I agree with having consistency and I like the `_lib` pattern. 

I added the `_binary` so TAP would find and exercise the rule in the build. Should we adopt this pattern as an additional rule for tools?

Reply:
TAP doesn't look for _targets_ ending in `_binary`, it looks for _rule types_ ending in `_binary`.

So this target will be tested because it is a `java_binary`, not because it's name is `foo_binary`.

Reply:
Ah! Thanks for clearing that up.